e2e2/test/cases/nvidia/manifests/mpi-job-pytorch-training-multi-node.yaml

```diff
@@ -5,26 +5,21 @@ metadata:
 spec:
   slotsPerWorker: 8
   runPolicy:
+    # it may take a bit for the workers to get ready (the container image is heavy)
+    # and we don't want the launcher to reach it's CrashLoopBackoff limit in the meantime
+    backoffLimit: 20
     cleanPodPolicy: Running
   mpiReplicaSpecs:
     Launcher:
       replicas: 1
       template:
          spec:
           restartPolicy: OnFailure
-          initContainers:
-          - image: public.ecr.aws/w6p6i9i7/aws-efa-nccl-rdma:22.03-pt-py3
-            name: init
-            command: ["sh", "-c", "sleep 5"]
           containers:
-          - image: public.ecr.aws/w6p6i9i7/aws-efa-nccl-rdma:22.03-pt-py3
+          - image: TODO
```

Comment:
Temporary until there's an build pipeline set up for the image added in this PR.